packages/postgres/pg-queue.ts

```diff
@@ -234,7 +234,7 @@ export class PgQueueRunner implements QueueRunner {
   constructor({
     adapter,
     workerId,
-    maxTimeoutSec = 10 * 60,
+    maxTimeoutSec = 5 * 60,
```

Comment:
i made this match the longest timeout that we use so we can. This is the value that the job will use internally to try to timeout (which it can't always do, e.g. busy waits that can only be interrupted externally)